package.json

```diff
@@ -29,7 +29,7 @@
   "homepage": "https://github.com/scality/backbeat#readme",
   "dependencies": {
     "@hapi/joi": "^15.1.0",
-    "arsenal": "scality/Arsenal#5f66ee9",
+    "arsenal": "scality/Arsenal#41b3bab",
```

Comment:
There will be a new git hash after https://github.com/scality/Arsenal/pull/1439 which will need to be applied to all branches.